go.mod

```diff
@@ -12,8 +12,40 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-go 1.18
+go 1.19
 
-replace golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd => golang.org/x/net v0.0.0-20220923203811-8be639271d50
+replace golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd => golang.org/x/net v0.10.0
+
+replace golang.org/x/net v0.0.0-20190620200207-3b0461eec859 => golang.org/x/net v0.10.0
+
+replace golang.org/x/net v0.0.0-20210226172049-e18ecbb05110 => golang.org/x/net v0.10.0
+
+replace golang.org/x/net v0.0.0-20220722155237-a158d28d115b => golang.org/x/net v0.10.0
+
+replace golang.org/x/net v0.0.0-20220923203811-8be639271d50 => golang.org/x/net v0.10.0
+
+replace golang.org/x/net v0.6.0 => golang.org/x/net v0.10.0
+
+replace golang.org/x/sys v0.0.0-20210615035016-665e8c7367d1 => golang.org/x/sys v0.8.0
+
+replace golang.org/x/sys v0.0.0-20220728004956-3c1f35247d10 => golang.org/x/sys v0.8.0
+
+replace golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a => golang.org/x/sys v0.8.0
+
+replace golang.org/x/sys v0.0.0-20201119102817-f84b799fce68 => golang.org/x/sys v0.8.0
+
+replace golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a => golang.org/x/sys v0.8.0
+
+replace golang.org/x/sys v0.0.0-20220722155257-8c9f86f7a55f => golang.org/x/sys v0.8.0
+
+replace golang.org/x/sys v0.5.0 => golang.org/x/sys v0.8.0
+
+replace golang.org/x/text v0.3.0 => golang.org/x/text v0.9.0
+
+replace golang.org/x/text v0.3.3 => golang.org/x/text v0.9.0
+
+replace golang.org/x/text v0.7.0 => golang.org/x/text v0.9.0
+
+replace golang.org/x/text v0.3.7 => golang.org/x/text v0.9.0
```

Comment:
Currently only v0.3.7 of golang.org/x/text is listed in `go.sum`. Why do we need all these replacements?

Reply:
Same goes for x/sys and x/net, I guess.

Reply:
It looks like after adding these replace statements the go.sum was condensed – the old list showed these versions. Should I remove these replace statements after updating go.sum?

Reply:
Are you sure you weren't looking at an amended list? If you look at the `go.sum` file on the `main` branch, [only v0.3.7 of golang.org/x/text is listed](https://github.com/cyberark/summon-aws-secrets/blob/main/go.sum#L22). I wonder if we could get away with only one of these replace statements.

Reply:
Good call - makes sense.